content/influxdb/v2.0/query-data/influxql.md

```diff
@@ -16,46 +16,101 @@ related:
+
+ - organization (`organization` or `organization_id`)
+ - target bucket (`bucket_id`)
+ - database and retention policy to map to bucket (`database` and `retention_policy`)
+
+```sh
+curl -XPOST https://cloud2.influxdata.com/api/v2/dbrps \
+  -H "Authorization: Token YourAuthToken" \
+  -H 'Content-type: application/json' \
+  -d '{
+       "bucket_id": "12ab34cd56ef",
+       "database": "example-db",
+       "default": true
+       "organization": "example-org",
+       "organization_id": "example-org",
+       "retention_policy": "example-rp",
+      }'
+```
+
+After you've verified the bucket is mapped, query the bucket using the `query` 1.x compatibility endpoint.
+
+## Query a mapped bucket with InfluxQL
+
 The [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/) supports
 all InfluxDB 1.x client libraries and integrations in InfluxDB 2.0.
 
-Provide the following:
+To query a mapped bucket with InfluxQL, use the `/query` 1.x compatibility endpoint (see CURL example below), and include the following in your request:
 
 - InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)
   _(See [compatibility API authentication](/influxdb/v2.0/reference/api/influxdb-1x/#authentication))_
 - **db query parameter**: 1.x database to query
-- **rp query parameter**: 1.x retention policy to query
+- **rp query parameter**: 1.x retention policy to query; if no retention policy is specified, InfluxDB uses the default retention policy for the specified database.
 - **q query parameter**: InfluxQL query
 
 {{% note %}}
 **URL-encode** the InfluxQL query to ensure it's formatted correctly when submitted to InfluxDB.
 {{% /note %}}
 
 ```sh
-curl -G https://cloud2.influxdata.com/query?db=db&rp=rp \
+curl -G https://cloud2.influxdata.com/query?database=MyDB&retention_policy=MyRP \
   -H "Authorization: Token YourAuthToken" \
   --data-urlencode "q=SELECT used_percent FROM mem WHERE host=host1"
 ```
 
 By default, the `/query` compatibility endpoint returns results in **JSON**.
 To return results as **CSV**, include the `Accept: application/csv` header.
 
-## Database and retention policy mapping
-InfluxDB 2.0 combines the 1.x concept of [databases](/influxdb/v1.8/concepts/glossary/#database)
-and [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
-into [buckets](/influxdb/v2.0/reference/glossary/#bucket).
-To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
-policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
-_See [DBRP mapping](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) for more information._
-
 ## InfluxQL support
-InfluxQL in InfluxDB 2.0 supports **read-only** queries.
+
+InfluxDB 2.0 supports InfluxQL **read-only** queries. See supported and unsupported queries below.
```

Comment:
Let's add:
To learn more about InfluxQL, see [Influx Query Language (InfluxQL)](/influxdb/v1.8/query_language/).